### ISSUE TITLE
Protocol: TCP/UDP option

### ISSUE THREAD

**Issue by Blurayx** [newloran2/emplexer#48]
Is it possible to have a TCP/UDP option for NFS in emplexer.
I have  a Dune Base 3D. When I setup a network folder with TCP enabled I do not get any stuttering.
The stuttering only happens with large files.

![1](https://f.cloud.github.com/assets/2873112/563938/ef0d7688-c528-11e2-86d6-f296282631b1.jpg)

**Comment by newloran2** [comment]
th emplexer use udp by default, to use tcp use in nfs setup nfs-tcp instead nfs.

**Comment by Blurayx** [comment]
Hi newloran2,
Sorry I don't understand, is there a TCP setting in emplexer?

**Comment by newloran2** [comment]
yes, in Nfs Advanced configuration, emplexer configure automatic your nfs like this: 
nfs://192.168.1.1:/path
this configuration is equivalent to nfs-udp to use tcp configure like this:
nfs-tcp://192.168.1.1:/path

**Comment by Blurayx** [comment]
when I set emplexer to nfs-tcp it defaults back to the original setting

**Comment by Blurayx** [comment]
Network folder settings
As you can see TCP offers a much higher bitrate with my synology 412+ setup.

![1](https://f.cloud.github.com/assets/2873112/564369/b2a2a25c-c55c-11e2-9d16-06b2f4b87229.jpeg)
![2](https://f.cloud.github.com/assets/2873112/564370/b70529d2-c55c-11e2-828d-bda2efd7b931.jpeg)

**Comment by Blurayx** [comment]
Hi newloran2,

Have you had anytime to fined out why emplexer can not keep the tcp settings.

Thanks

**Comment by Blurayx** [comment]
UPDATE
After a reinstall and system reset emplexer now accepts the settings but I still get stuttering.
The  advance setting say nfs, but on my dune it displays nfs-tcp.

**Comment by Blurayx** [comment]
Hi newloran2

Not sure if you received my email, but just to let you know, the emplexer tcp fix you kindly sent me works.
the speed difference with tcp enabled on emplexer with my setup is amazing.
Setup:
synology 412+
TP-LINK 500mbps(Home Plug)
Note: Not sure if it makes any difference but you can now enable synology's MTU settings 

Thank you

**Comment by lastiko** [comment]
Working on Qnap too
No problem to read ISO Bluray ( BDlite ) on Dune 

The only thing i see is , i can"t read Music .... Thats all

Really good job , if u need help ...

**Comment by Blurayx** [comment]
Hi lastiko,
Were you also having problems with the udp settings?

**Comment by lastiko** [comment]
No problem with UDP 
i'm using SMB , all work very fine 
Only question is , do you think u will do something for Music ?

Good Job Sir

**Comment by newloran2** [comment]
lastiko,

Music and photos don't work at this moment, but i will develop this features soon.

**Comment by Blurayx** [comment]
Hi newloran2,
Update
I've noticed when I enable udp in the network folder it interacts with  emplexer, causing it to play in udp mode.
